hypershift-operator/controllers/platform/aws/controller.go

```diff
@@ -521,10 +521,36 @@ func (r *AWSEndpointServiceReconciler) delete(ctx context.Context, awsEndpointSe
 		ServiceIds: []*string{aws.String(serviceID)},
 	})
 	if err != nil {
-		if awsErr, ok := err.(awserr.Error); ok {
-			return false, errors.New(awsErr.Code())
+		log.Error(err, "Failed to delete vpc endpoint service configuration", "serviceID", serviceID)
```

Comment:
to reduce the number of code paths and complexity, we could also do this unconditionally not only if error?

Reply:
In the "happy" path (and even not so happy path), we normally don't need to do this. I didn't want to add the additional calls unless strictly necessary.